### DISCUSSION TITLE
Business Email Problem

### DISCUSSION THREAD

**Discussion by arshad657** [vercel/community#3014] [Help] [answered]
### Summary

I chose vercel as my hosting provider. But after deploying it, my business email is not getting messages from third party website.
I integrated my business mail in a website to get messages from there  (I used fromspree for that). But , I am not receiving messages from there. My configuration is fine as I get messages for other email addresses, but when I add my business mail, it doesn't work. I bought my domain from namecheap and contacted with them regarding this issue.

### Example

receiving mails that are directly composed.

Not receiving mails that are coming from the third party side.

### Steps to Reproduce

![Screenshot (146)](https://github.com/vercel/community/assets/70918172/050cb092-3426-47f3-b777-ac09b194980d)

**Comment by swarnava** [comment, marked as answer]
Hi there,

You need to setup desired MX record in order to receive email in your mailbox. For example, here is the instruction for Google Workspace: https://support.google.com/a/answer/174125?hl=en#zippy=%2Ci-signed-up-before.

Please contact to your email hosting provider for more information on this. As Vercel don't provide mail server, we unfortunately don't have more context to troubleshoot this.
